src/scene/materials/standard-material.js

```diff
@@ -227,19 +257,41 @@ let _params = new Set();
  * @property {Texture|null} refractionMap The map of the refraction visibility.
  * @property {number} refractionMapUv Refraction map UV channel.
  * @property {Vec2} refractionMapTiling Controls the 2D tiling of the refraction map.
- * @property {Vec2} refractionMapOffset Controls the 2D offset of the refraction map. Each component is
- * between 0 and 1.
+ * @property {Vec2} refractionMapOffset Controls the 2D offset of the refraction map. Each component
+ * is between 0 and 1.
  * @property {number} refractionMapRotation Controls the 2D rotation (in degrees) of the emissive
  * map.
  * @property {string} refractionMapChannel Color channels of the refraction map to use. Can be "r",
  * "g", "b", "a", "rgb" or any swizzled combination.
+ * @property {boolean} refractionVertexColor Use mesh vertex colors for refraction. If
+ * refraction map is set, it will be be multiplied by vertex colors.
+ * @property {boolean} refractionVertexColorChannel Vertex color channel to use for refraction.
+ * Can be "r", "g", "b" or "a".
  * @property {number} refractionIndex Defines the index of refraction, i.e. The amount of
  * distortion. The value is calculated as (outerIor / surfaceIor), where inputs are measured
  * indices of refraction, the one around the object and the one of its own surface. In most
  * situations outer medium is air, so outerIor will be approximately 1. Then you only need to do
  * (1.0 / surfaceIor).
  * @property {boolean} useDynamicRefraction Enables higher quality refractions using the grab pass
  * instead of pre-computed cube maps for refractions.
+ * @property {number} thickness The thickness of the medium, only used when useDynamicRefraction
+ * is enabled.
```

Comment:
So for a 1x1x1 (unit) cube, you'd set this to 1?

Reply:
I guess we should specify which unit this is in, but yeah, it's whatever unit the scale of the model is in. 1 would mean 1 world unit, and 2 would mean 2, and so forth. I'll add unit denotations in the docs as well.

Reply:
But yeah, for a unit sphere of size 1, it would make sense to make this 1. However, it also scales with the object using the object transform, so if you set it to 1 and scale the object up, you can expect the thickness to conform to the final transformation. So this property is reusable for objects with different scale, no need to create a material for each.